Issue building python wrapper with CGNS support (problems with static/dynamic HDF5 dependencies)

**Describe the bug**

I am unable to complete the compilation of SU2 v7.3.0 on Fedora 35 using the GCC compiler.
There is a problem when trying to link with the system's libz library.

```
/usr/bin/ld: /usr/lib64/libz.a(deflate.o): warning: relocation against `z_errmsg' in read-only section `.text'
/usr/bin/ld: /usr/lib64/libz.a(zutil.o): relocation R_X86_64_PC32 against symbol `z_errmsg' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: bad value
collect2: error: ld returned 1 exit status

```
**To Reproduce**

I configured the source code with the commands:

```
module load mkl/latest
module load mpi/openmpi-x86_64

CFLAGS="-mtune=native -march=native -fPIC" CXXFLAGS="-mtune=native -march=native -fPIC" LDFLAGS="-fPIC" CC=$(which mpiicc) CC=$(which mpicc) CXX=$(which mpicxx) ./meson.py --prefix=/home/panos/Applications/SU2 --optimization=3 -Denable-cgns=true -Denable-mkl=true -Denable-autodiff=true -Denable-directdiff=true -Denable-pywrapper=true -Dwith-mpi=enabled build

Message: -------------------------------------------------------------------------
         |    ___ _   _ ___                                                      |
         |   / __| | | |_  )   Release 7.3.0 "Blackbird"                         |
         |   \__ \ |_| |/ /                                                      |
         |   |___/\___//___|   Meson Configuration Summary                       |
         |                                                                       |
         -------------------------------------------------------------------------

         Option          Value
         ---------------------
         TecIO:          true
         CGNS:           true
         AD (reverse):   true
         AD (forward):   true
         Python Wrapper: true
         Intel-MKL:      true
         OpenBlas:       false
         PaStiX:         false
         Mixed Float:    false
         libROM:         false

         Please be sure to add the $SU2_HOME and $SU2_RUN environment variables,
         and update your $PATH (and $PYTHONPATH if applicable) with $SU2_RUN

         Based on the input to this configuration, add these lines to your .bashrc file:

         export SU2_RUN=/home/panos/Applications/SU2/bin
         export SU2_HOME=/home/panos/Development/SU2/SU2_SRC
         export PATH=$PATH:$SU2_RUN
         export PYTHONPATH=$PYTHONPATH:$SU2_RUN

         Use './ninja -C build install' to compile and install SU2

Build targets in project: 25

SU2 7.3.0 "Blackbird"

  User defined options
    optimization     : 3
    prefix           : /home/panos/Applications/SU2
    enable-autodiff  : true
    enable-cgns      : true
    enable-directdiff: true
    enable-mkl       : true
    enable-pywrapper : true
    with-mpi         : enabled

Found ninja-1.10.0.git at /home/panos/Development/SU2/SU2_SRC/ninja
```

When compiling the process fails when trying to link with the libz library:

```
[1357/1358] Linking target SU2_PY/pySU2/_pysu2ad.so
FAILED: SU2_PY/pySU2/_pysu2ad.so
/usr/lib64/openmpi/bin/mpicxx  -o SU2_PY/pySU2/_pysu2ad.so SU2_PY/pySU2/_pysu2ad.so.p/meson-generated_pySU2ad.cxx.o SU2_CFD/src/libSU2core_AD.a.p/definition_structure.cpp.o SU2_CFD/src/libSU2core_AD.a.p/python_wrapper_structure.cpp.o SU2_CFD/src/libSU2core_AD.a.p/CMarkerProfileReaderFVM.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CFluidModel.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CIdealGas.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CPengRobinson.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CVanDerWaalsGas.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CNEMOGas.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CMutationTCLib.cpp.o SU2_CFD/src/libSU2core_AD.a.p/fluid_CSU2TCLib.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_COutputFactory.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CAdjElasticityOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CAdjFlowOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CAdjFlowCompOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CAdjFlowIncOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CAdjHeatOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CBaselineOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CMultizoneOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CElasticityOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CFlowCompOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CFlowCompFEMOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CFVMOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CFlowOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CFlowIncOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CHeatOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CMeshOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_CNEMOCompOutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_output_physics.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_COutput.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_output_structure_legacy.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParallelDataSorter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CFVMDataSorter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CFEMDataSorter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSurfaceFEMDataSorter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSurfaceFVMDataSorter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParallelFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParaviewFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParaviewBinaryFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CTecplotFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CTecplotBinaryFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CCSVFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSTLFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSU2FileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSU2BinaryFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParaviewXMLFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CParaviewVTMFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CSU2MeshFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_filewriter_CCGNSFileWriter.cpp.o SU2_CFD/src/libSU2core_AD.a.p/output_tools_CWindowingTools.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CIncNSVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CTransLMVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CAdjEulerVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CHeatVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CTurbVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CScalarVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CSpeciesVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CAdjNSVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CBaselineVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CDiscAdjFEABoundVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CDiscAdjMeshBoundVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CFEABoundVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CDiscAdjVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CTurbSAVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CFEAVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CMeshVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CMeshBoundVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CMeshElement.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CRadVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CRadP1Variable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CTurbSSTVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CNSVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CAdjTurbVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CFlowVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CIncEulerVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CEulerVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CNEMOEulerVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CNEMONSVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/variables_CSobolevSmoothingVariable.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CSolverFactory.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CAdjEulerSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CAdjNSSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CAdjTurbSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CBaselineSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CBaselineSolver_FEM.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CDiscAdjFEASolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CDiscAdjMeshSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CRadSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CRadP1Solver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CDiscAdjSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CEulerSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CFEASolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CFEM_DG_EulerSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CFEM_DG_NSSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CHeatSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CIncEulerSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CIncNSSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CMeshSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CNEMOEulerSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CNEMONSSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CNSSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CTemplateSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CSpeciesSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CTransLMSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CTurbSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CTurbSASolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CTurbSSTSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CFEASolverBase.cpp.o SU2_CFD/src/libSU2core_AD.a.p/solvers_CGradientSmoothingSolver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_CNumerics.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_template.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_transition.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_heat.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_radiation.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_roe.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_fds.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_fvs.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_cusp.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_hllc.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_ausm_slau.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_convection_centered.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_flow_diffusion.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_flow_flow_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_CNEMONumerics.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_lax.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_roe.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_ausm.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_ausmplusup2.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_ausmpwplus.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_convection_msw.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_NEMO_diffusion.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_NEMO_NEMO_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_continuous_adjoint_adj_convection.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_continuous_adjoint_adj_diffusion.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_continuous_adjoint_adj_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_scalar_scalar_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_turbulent_turb_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_species_species_sources.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_elasticity_CFEAElasticity.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_elasticity_CFEALinearElasticity.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_elasticity_CFEANonlinearElasticity.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_elasticity_nonlinear_models.cpp.o SU2_CFD/src/libSU2core_AD.a.p/numerics_CGradSmoothing.cpp.o SU2_CFD/src/libSU2core_AD.a.p/.._include_numerics_simd_CNumericsSIMD.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_CInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_cfd_CConservativeVarsInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_cfd_CMixingPlaneInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_cfd_CSlidingInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_cht_CConjugateHeatInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_fsi_CDisplacementsInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_fsi_CFlowTractionInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/interfaces_fsi_CDiscAdjFlowTractionInterface.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CMultizoneDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CSinglezoneDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CDiscAdjMultizoneDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CDiscAdjSinglezoneDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/drivers_CDummyDriver.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CIntegration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CIntegrationFactory.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CSingleGridIntegration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CMultiGridIntegration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CNewtonIntegration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CStructuralIntegration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/integration_CFEM_DG_Integration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CIterationFactory.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CAdjFluidIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CDiscAdjFluidIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CDiscAdjHeatIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CDiscAdjFEAIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CFEAIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CFEMFluidIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CFluidIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CHeatIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/iteration_CTurboIteration.cpp.o SU2_CFD/src/libSU2core_AD.a.p/limiters_CLimiterDetails.cpp.o -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 -shared -fPIC -Wl,--start-group -Wl,-soname,_pysu2ad.so -fPIC -mtune=native -march=native -fPIC -Wl,-rpath,/usr/lib64/openmpi/lib -Wl,-rpath-link,/usr/lib64/openmpi/lib Common/src/libSU2CommonAD.a externals/cgns/libcgns.a externals/cgns/hdf5/libsu2hdf5.a externals/metis/libmetis.a externals/parmetis/libparmetis.a externals/tecio/teciompisrc/libteciompi.a /usr/lib64/libpython3.10.so /usr/lib64/libz.a -lm -ldl -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags /usr/lib64/openmpi/lib/libmpi.so -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags /usr/lib64/openmpi/lib/libmpi_cxx.so -Wl,--start-group /opt/intel/oneapi/mkl/latest/lib/pkgconfig/../../lib/intel64/libmkl_intel_lp64.a /opt/intel/oneapi/mkl/latest/lib/pkgconfig/../../lib/intel64/libmkl_sequential.a /opt/intel/oneapi/mkl/latest/lib/pkgconfig/../../lib/intel64/libmkl_core.a -Wl,--end-group -lpthread -Wl,--end-group -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,--start-group -Wl,--end-group -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags
/usr/bin/ld: /usr/lib64/libz.a(deflate.o): warning: relocation against `z_errmsg' in read-only section `.text'
/usr/bin/ld: /usr/lib64/libz.a(zutil.o): relocation R_X86_64_PC32 against symbol `z_errmsg' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: bad value
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.

```

I do use the -fPIC option for the compilation process but even if I remove it from the configuration execution line I still get the same linking error.

**Desktop (please complete the following information):**

 - OS: Linux (Fedora 35)
 - C++ 11.2.1 20220127 (Red Hat 11.2.1-9) (GCC)
 - MPI implementation and version: OpenMPI v4.1.1
 - SU2 Version: v7.3.0

--- a/externals/cgns/hdf5/meson.build
+++ b/externals/cgns/hdf5/meson.build
@@ -25,8 +25,8 @@ hdf5_conf_data.set('version', '1.12.1')
 ext_deps = []
 
 
-opt_zlib = dependency('zlib', required: false, static: true)
-opt_szip = dependency('szip', required: false, static: true)
+opt_zlib = dependency('zlib', required: false, static: false)
+opt_szip = dependency('szip', required: false, static: false)
 
 if opt_zlib.found() and cc.has_header('zlib.h')
   if cc.has_function('inflate', dependencies: opt_zlib, prefix: '#include <zlib.h>')

